### PR TITLE
fix(presets): migrate renamed monorepos

### LINKED ISSUE #11318: Migrate breaking preset names in v26
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

_No response_

### If you're self-hosting Renovate, tell us what version of Renovate you run.

_No response_

### Describe the bug

Described here: https://github.com/renovatebot/renovate/pull/10243#issuecomment-900917024

### Relevant debug logs

See https://github.com/renovatebot/renovate/pull/10243#issuecomment-900917024

### Have you created a minimal reproduction repository?

No reproduction repository

### PR DESCRIPTION
**Pull request by rarkins**
## Changes:

Adds migrations for renamed monorepo presets.

## Context:

Closes #11318

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by rarkins** [comment]
@HonkingGoose can you double check that I caught every change?

File: lib/config/presets/common.ts [hunk 1]
```diff
@@ -23,3 +23,39 @@ export const removedPresets = {
   'helpers:oddIsUnstable': null,
   'helpers:oddIsUnstablePackages': null,
 };
+
+const renamedMonorepos: Record<string, string> = {
+  'arcus event-grid': 'arcus.event-grid',
+  'arcus security': 'arcus.security',
+  'arcus messaging': 'arcus.messaging',
+  'arcus observability': 'arcus.observability',
+  'arcus webap': 'arcus.webap',
```
Comment (HonkingGoose) [inline lib/config/presets/common.ts:32]:
```suggestion
  'arcus webapi': 'arcus.webapi',
```

I think a typo slipped in, my original PR https://github.com/renovatebot/renovate/pull/10243/files made the change from `arcus webapi` -> `arcus.webapi`

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Still checking the others, but we can fix this one now.

**Review by HonkingGoose — COMMENTED** [review]
I can confirm we have migrated the changes that I made in my _original_ PR: https://github.com/renovatebot/renovate/pull/10243/files

I don't think we edited it any further in the `feat: v26` PR: https://github.com/renovatebot/renovate/pull/10995/files#diff-79c717b40c9bee60d2c9256ae4944499e287678a4880b0c457ecdd1b8e111466

But might be good to run a diff between these two somehow to confirm we didn't make any further changes????

**Comment by rarkins** [comment]
I'd prefer to merge this ASAP if it satisfies both our checks

**Comment by HonkingGoose** [comment]
I ran a diff between my original PR and the `feat: v26` PR, and we didn't make any more edits to existing monorepos. 

We did _add_ new monorepos in the `feat: v26` PR, but new monorepos don't need a migration, as they're _new_. 😄 

I'd say we're good to merge now.

**Review by HonkingGoose — APPROVED** [review]

**Review by viceice — APPROVED** [review]
Hopefully you've found all. 😅

**Pull request merged**